pkg/virtctl/imageupload/imageupload.go

```diff
@@ -926,3 +951,56 @@ func handleEventErrors(client kubecli.KubevirtClient, pvcName, dvName, namespace
 
 	return nil
 }
+
+func createNewDataSource(client kubecli.KubevirtClient, name, namespace string, labels map[string]string) error {
+	ds := &v1beta1.DataSource{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+			Labels:    labels,
+		},
+		Spec: v1beta1.DataSourceSpec{
+			Source: v1beta1.DataSourceSource{
+				PVC: &v1beta1.DataVolumeSourcePVC{
+					Name:      name,
+					Namespace: namespace,
+				},
+			},
+		},
+	}
+
+	if _, err := client.CdiClient().CdiV1beta1().DataSources(namespace).Create(context.Background(), ds, metav1.CreateOptions{}); err != nil {
```

Comment:
Let this func always return err and move the cmd.Printf outside this func to the place where it is called.

Reply:
Sure thing.